src/views/templates/actions/VirtualMachineTemplatesActions.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Action } from '@openshift-console/dynamic-plugin-sdk';
+import { Dropdown, DropdownItem, DropdownPosition, KebabToggle } from '@patternfly/react-core';
+
+import useVirtualMachineTemplatesActions from './hooks/useVirtualMachineTemplatesActions';
+
+type VirtualMachineTemplatesActionsProps = { template: V1Template };
+
+const VirtualMachineTemplatesActions: React.FC<VirtualMachineTemplatesActionsProps> = ({
+  template,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const actions = useVirtualMachineTemplatesActions(template);
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const handleClick = (action: Action) => {
+    if (typeof action?.cta === 'function') {
+      action?.cta();
+      setIsOpen(false);
+    }
+  };
+
+  const disabledActionInfo = (actionId: string): string => {
+    switch (actionId) {
+      case 'edit-boot-source':
+        return t('Boot source cannot be edited for Red Hat templates');
+      case 'delete-template':
+        return t('Red Hat templates cannot be deleted');
```

Comment:
Red Hat provided templates can be deleted, the templates you can't delete are the common templates (owned by SSP operator)

Reply:
I followed the design I was given. If we need to change this, we need the design update, too.

Reply:
makes sense to me to